### PR TITLE
test: add unit tests for extractCustomFields function

### PR DESCRIPTION
**Pull request by yashasvibajpai**
## Description of the change

> Added unit tests for `extractCustomFields` function

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by utsabc — APPROVED** [review]

File: test/__tests__/__integration_util_test__/data/extractCustomFields.json [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      "conversion_source": "RudderStack",
+      "action": "conv.add_to_cart",
+      "checkout_id": "12345",
+      "product_url": "http://www.yourdomain.com/products/red-t-shirt",
+      "product_name": "Red T-shirt",
+      "externalId": "rbUid"
+    }
+  },
+  {
+    "description": "Check for message fields absent from the exclusion list and adding them to payload object - 2",
```
Comment (sanpj2292) [inline test/__tests__/__integration_util_test__/data/extractCustomFields.json:92]: [Nitpick]
Let's exactly describe what we're trying to see in the test-case name itself & avoid using numbers to include which test-case it is as much as possible

**Review by sanpj2292 — APPROVED** [review]

File: test/__tests__/__integration_util_test__/data/extractCustomFields.json (hunk 1 above)
Comment (yashasvibajpai) [reply test/__tests__/__integration_util_test__/data/extractCustomFields.json:92]: The function has a very specific usecase, hence in order to have more test data numbered the case.

Comment (sanpj2292) [reply test/__tests__/__integration_util_test__/data/extractCustomFields.json:92]: You can mention what fields you intend to probably extract in the test-case, with that you can directly look at how each test-case is different from each other & also provide some text to indicate what we expect to happen through the test-case.

Comment (sanpj2292) [reply test/__tests__/__integration_util_test__/data/extractCustomFields.json:92]: Anyways this is more like nitpicking(non-blocking) comment, please try to follow this in upcoming test-case naming conventions and proceed. For now, will be resolving this comment

**Pull request merged**